internal/commands/result.go

```diff
@@ -1429,6 +1499,107 @@ func parseGlSastVulnerability(result *wrappers.ScanResult, glSast *wrappers.GlSa
 	return glSast
 }
 
+func parseGlDependencyVulnerability(result *wrappers.ScanResult, glDependencyResult *wrappers.GlDependencyResultsCollection) *wrappers.GlDependencyResultsCollection {
+
+	if result.ScanResultData.ScaPackageCollection != nil {
+
+		glDependencyResult.Vulnerabilities = append(glDependencyResult.Vulnerabilities, wrappers.GlDepVulnerabilities{
+			Id:          result.ID,
+			Name:        result.VulnerabilityDetails.CveName,
+			Description: result.Description,
+			Severity:    cases.Title(language.English).String(result.Severity),
+			Solution:    result.ScanResultData.RecommendedVersion,
+			Identifiers: collectDependencyPackageData(result),
+			Links:       collectDependencyPackageLinks(result),
+			TrackingDep: wrappers.TrackingDep{
+				Items: collectDependencyPackageItemsDep(result),
+			},
+			Flags: make([]string, 0),
+			LocationDep: wrappers.GlDepVulnerabilityLocation{
+				File: *(result.ScanResultData.ScaPackageCollection.Locations[0]),
+				Dependency: wrappers.DependencyLocation{
+					Package:                   wrappers.PackageName{Name: result.ScanResultData.PackageIdentifier},
+					DependencyLocationVersion: "",
```

Comment:
you can delete these empty properties, I think it is the same if you don't provide them